test/system/500-networking.bats

```diff
@@ -430,6 +430,7 @@ load helpers
     is "$output" "[${cid:0:12}]" "short container id in network aliases"
 
     run_podman network disconnect $netname $cid
+    is "" "" "Output should be empty (no errors)"
```

Comment:
Tautological: should be `is "$output" ""`

Reply:
I think I should call it a day, thanks